features/FEATURE_LWIP/lwip-interface/CONTRIBUTING.md

```diff
@@ -0,0 +1,129 @@
+
+    `e.g. git pull lwip STABLE-2_0_2_RELEASE_VER`
+
+4. Push tag to mbed OS lwip fork repository to keep fork in sync.
+
+    `git push origin <tag>`
+
+    `e.g. git push origin STABLE-2_0_2_RELEASE_VER`
+
+5. Create branch for release. Use same naming convention as previous branches.
+
+    `git branch <branch name> <tag>`
+
+    `e.g. git branch mbed-os-lwip-stable-2_0_2 STABLE-2_0_2_RELEASE_VER`
+
+5. Push branch to mbed OS lwip fork repository to keep fork in sync.
+
+    `git push origin <branch name>`
+
+    `e.g. git push origin mbed-os-lwip-stable-2_0_2`
+
+6. Fetch previous prefixed branch from mbed OS lwip fork repository.
+
+    `git fetch <remote name> <remote branch>:<local branch>` 
+
+    `e.g. git fetch origin mbed-os-lwip-stable-2_0_1-prefixed:mbed-os-lwip-stable-2_0_1-prefixed`
+
+7. Checkout latest prefixed branch.
+
+    `git checkout <prefixed branch>`
+
+    `e.g. git checkout mbed-os-lwip-stable-2_0_1-prefixed`
+
+8. Merge new branch to prefixed branch.
+
+    `git merge <branch name>`
+
+    `e.g. git merge mbed-os-lwip-stable-2_0_2`
+
+9. Check if there are any new or colliding files.
+
+    If there are new c-files rename those with `lwip_` prefix and make a commit about 
+    changes to prefixed branch. Also check that lwip stack header file names does not 
+    collide with headers under mbed OS directory tree. If there are collisions rename the
+    colliding lwip headers with `lwip_` prefix.
+
+    If needed rename files and make commit. In case header files are renamed update also
+    c-files that include them.
+
+    `git mv <old filename> <new filename>`
+    
+    `git commit`    
+
+10. Rename local prefixed branch to new version (do not change branch names on remotes)
+
+    `git branch -m <new prefixed branch>`
+
+    `e.g. git branch -m mbed-os-lwip-stable-2_0_2-prefixed`
+
+11. Push the new local prefixed branch to mbed OS lwip fork repository
+
+    `git push origin <new prefixed branch>`
+
+    `e.g. git push origin mbed-os-lwip-stable-2_0_2-prefixed`
+
+### Merging prefixed release to mbed OS repository
+
+1. Clone mbed OS repository.
+
+    `git clone git@github.com:ARMmbed/mbed-os.git`
+
+2. Go to mbed OS root directory and add mbed OS lwip fork repository as remote. Fetch branches from fork.
+
+    `git remote add <remote name> https://github.com/ARMmbed/lwip.git`
+
+    `e.g. git remote add lwip-fork https://github.com/ARMmbed/lwip.git`
+
+    `git fetch <remote name>`
+
+    `e.g. git fetch lwip-fork`
+
+3. Do subtree pull for lwip prefixed branch in mbed OS root directory.
+
+    `git subtree pull --squash -P features/FEATURE_LWIP/lwip-interface/lwip <remote name> <new prefixed branch> -m "<commit message>"`
+
+    `e.g. git subtree pull --squash -P features/FEATURE_LWIP/lwip-interface/lwip lwip-fork mbed-os-lwip-stable-2_0_2-prefixed -m "Merged lwip 2.0.2 stable"`
+
+    After this step there should be a new commit visible is mbed OS master branch that contains
```

Comment:
This implies that you want "in mbed OS the master branch". I think " in the mbed OS master branch" is the more common form.